imagetest/cmd/manager/main.go

```diff
@@ -36,6 +36,7 @@ var (
 	filter        = flag.String("filter", "", "only run tests matching filter")
 	exclude       = flag.String("exclude", "", "skip tests matching filter")
 	machineType   = flag.String("machine_type", "", "machine type to use for test instances")
+	osUnderTest   = flag.String("os", "linux", "operating system that tests will be run on")
```

Comment:
Is there any reason not to use `runtime.GOOS` to determine the OS.

Also instead of passing the value, I'd suggest having the test do the detection as it shouldn't need to be passed.

Reply:
we discussed this offline and i think the better strategy is to resolve compilation errors with OS-specific build constraints but leave test selection as usual.